ui/custom-elements.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
What does this file do? I haven't seen it before

Reply:
no idea, was included in the scaffold. should i delete?

Reply:
Yeah, we don't use this internally

Reply:
removed in upcoming pr